### PR TITLE
Update synthetic elections

### PR DESCRIPTION
**Pull request by JDziurlaj**
All:
- Stablizes `GeneratedDate` to fixed value

CVR:
- Adds property `CVRContestSelection::OptionPosition`
- Correctly calculates undervotes for contests controlled by a straight party contest
- Adds `CVRContest:Status` when contest is `undervoted` or `overvoted`

ERR:
 - Adds `OtherCounts::Overvotes` and `OtherCounts::Undervotes` at precinct level
 - Removes `VoteCounts` for precincts that do not participate in the contest
 - Changes `ElectionReport::Status` from `unoffical-partial` to `unofficial-complete`

### CODE REVIEW

File: test_data/real_world/ma/cambridge/2022/general/err_v2/err-2022-11-08-Massachusetts-Cambridge.xml
Comment (jungshadow) [inline test_data/real_world/ma/cambridge/2022/general/err_v2/err-2022-11-08-Massachusetts-Cambridge.xml:1]: I just realized that the `err` prefix might seem like an error file to some. I don't have a suggestion, but that just came to mind.

File: test_data/synthetic/gen-01/README.md [hunk 1]
```diff
@@ -39,18 +36,17 @@ several test elections that has been developed.
 <li><p>Vote for N of M: Yes</p></li>
 <li><p>Recall issues, with options: No</p></li>
 <li><p>Cumulative voting: No</p></li>
-<li><p>Ranked order voting: No</p></li>
-<li><p>Provisional or challenged ballots: Yes</p></li>
-<li><p>Early Voting: No</p></li>
+<li><p>Ranked choice voting: No</p></li>
```
Comment (jungshadow) [inline test_data/synthetic/gen-01/README.md:39]: Why is there a paragraph tag inside the list item? It's not wrong, per se, but seems unnecessary.

File: test_data/synthetic/gen-01/README.md [hunk 2]
```diff
@@ -39,18 +36,17 @@ several test elections that has been developed.
...
+<li></li>
```
Comment (jungshadow) [inline test_data/synthetic/gen-01/README.md:40]: Do these two additional empty list items get rendered?

File: test_data/synthetic/gen-01/README.md [hunk 3]
```diff
@@ -15,16 +15,13 @@ several test elections that has been developed.
 <table>
```
Comment (jungshadow) [inline test_data/synthetic/gen-01/README.md:15]: Some of the content here makes sense as a table, but this opening preamble doesn't.

File: test_data/synthetic/prim-01/README.md [hunk 4]
```diff
@@ -0,0 +1,6091 @@
+# NIST Voting Common Data Format Test Method – Test Data
```
Comment (jungshadow) [inline test_data/synthetic/prim-01/README.md:1]: Same comment as the previous file. There's a lot of information that doesn't need to be a part of the table.

File: test_data/synthetic/prim-03/README.md [hunk 5]
```diff
@@ -15,14 +15,11 @@ several test elections that has been developed.
 <table>
```
Comment (jungshadow) [inline test_data/synthetic/prim-03/README.md:15]: Same comment as before about the table.

**Review by jungshadow — CHANGES_REQUESTED** [review]
Impressive and excellent work, @JDziurlaj! Just a few stylistic bits that need a bit of cleanup (e.g., I imagine the table-based layout might have been a straight pandoc translation of the original synthetic Word doc).

**Comment by JDziurlaj** [comment]
I can unwrap outer Word table and most of them will get converted to normal GFM tables. However, the one showing the precinct / contest crosswalk will still render as HTML, as apparently GFM doesn't support column spans.

**Comment by JDziurlaj** [comment]
Removed outer table from ProV&V template. HTML tables were converted to GFM tables for test decks. Ballot definition remains in HTML due to need for colspans.

**Comment by jungshadow** [comment]
👍 LGTM

**Pull request merged**